cpp/serve/grammar/json_schema_converter.cc

```diff
@@ -23,6 +23,14 @@ namespace serve {
 
 using namespace tvm::runtime;
 
+// EMCC somehow cannot pickup operator overload from picojson.h, so we copy here.
+#ifdef COMPILE_MLC_WASM_RUNTIME
+inline std::ostream& operator<<(std::ostream& os, const picojson::value& x) {
```

Comment:
For some reason without this, EMCC would throw error complaining about invalid operands to binary expression:
![image](https://github.com/mlc-ai/mlc-llm/assets/53290280/2c5543d3-811e-4f10-8b8a-9c90e06ffc17)

If I define any other dummy functions in `picojson.h`, it would get picked up by EMCC; but this << overload function cannot be picked up. So I redefined it here, wrapped in `COMPILE_MLC_WASM_RUNTIME` so that it does not affect other flows.